Directory.Build.props

```diff
@@ -86,7 +86,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Nerdbank.GitVersioning" Version=" 2.1.65" PrivateAssets="all" />
+    <PackageReference Include="Nerdbank.GitVersioning" Version="3.1.91" PrivateAssets="all" />
```

Comment:
You changed the version here but you should also update it on the build.cake file:
https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/ae4000bcc6cd8eb235b9915a811bd5ea1097169a/build/build.cake#L23

Reply:
Yeah, I realized that after, I'll make the fix. Thanks!